code/modules/unit_tests/mapping.dm

```diff
@@ -18,3 +18,16 @@
 
 		TEST_FAIL(log_entry)
 
+
+/// Tests that [/datum/job/proc/get_default_roundstart_spawn_point] returns a landmark from all joinable jobs.
+/datum/unit_test/job_roundstart_spawnpoints
```

Comment:
Should make this its own file, I don't want this to be a catch all file for mapping problems